### PR TITLE
insignificant changes

### PR DESCRIPTION
**Pull request by tzaffi**
Breaking out small unrelated changes in #552 to make that PR (when ready to review) a bit easier to grok.

### CODE REVIEW

**Review by ahangsu — APPROVED** [review]

File: pyteal/ast/bytes.py [hunk 1]
```diff
@@ -1,10 +1,10 @@
-from typing import cast, overload, TYPE_CHECKING
```
Comment (tzaffi) [inline pyteal/ast/bytes.py:1]: sorting imports

File: pyteal/compiler/optimizer/optimizer.py [hunk 2]
```diff
@@ -1,8 +1,8 @@
 from typing import Final, Optional, Set
```
Comment (tzaffi) [inline pyteal/compiler/optimizer/optimizer.py:1]: sorting imports

File: pyteal/ir/tealconditionalblock.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
-from typing import Optional, List
```
Comment (tzaffi) [inline pyteal/ir/tealconditionalblock.py:1]: prefer `|` over `Optional`

File: scripts/generate_init.py [hunk 4]
```diff
@@ -104,7 +104,7 @@ def overwrite(regen: str):
                 "The __init__.pyi needs to be regenerated. Please run scripts/generate_init.py"
             )
             sys.exit(1)
-        print("No changes in __init__.py")
```
Comment (tzaffi) [inline scripts/generate_init.py:107]: correct the message

**Review by jasonpaulos — APPROVED** [review]

**Pull request merged**